.github/workflows/jpa_integration_ci.yml

```diff
@@ -38,20 +38,6 @@ jobs:
         maven-version: [ '3.8.6' ]
       fail-fast: false
     runs-on: ${{ matrix.os }}
-    services:
```

Comment:
Who starts the postgres now?
And while we're at it, can we just remove this CI job altogether? I've never seen it fail, and it will save some precious GHA resources.

Reply:
That's the best thing, as we now have Quarkus on the test classpath, it also brings its [dev services](https://quarkus.io/guides/databases-dev-services), which starts the DB via Testcontainers out of the box; no additional configuration or code is needed. 
One more benefit of such an approach is that you can run the tests with PostgreSQL locally without having to care about how to start the DB. For Testcontainers, you just need to have Docker on your laptop.

As for the CI check, yes, that's an option, but the `postgresql` profile still needs to be activated. If we want to test both H2 and PostgreSQL, we have to run the tests twice, so there has to be an extra step of any GHA CI anyway.

Reply:
I'm kinda questioning if we need the postgres check. New world, no DB matrix, no containers, isn't H2 in Quarkus enough?

Reply:
Also, the JPA CI doesn't do build chain, so it occasionally fails because of that. I'm starting to think of it as technical debt.

Reply:
Maybe you're right; I don't remember the PostgreSQL CI check failing for a good reason. The truth is we don't do anything spectacular regarding persistence so we to some extent duplicate the testing effort of Hibernate.

Reply:
I removed the PostgreSQL support and the related CI check, too.